### PR TITLE
feat/added Sample Collection timstamp

### PR DESCRIPTION
**Pull request by AnnotationGuyTest**
added timestamp for sample collection

### CODE REVIEW

**Review by ascheibal — APPROVED** [review]

File: src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 package app.coronawarn.quicktest.model;
 
+
```
Comment (mschulte-tsi) [inline src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java:23]: please remove

File: src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java [hunk 2]
```diff
@@ -30,6 +32,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
+
```
Comment (mschulte-tsi) [inline src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java:35]: please remove

File: src/main/java/app/coronawarn/quicktest/service/QuickTestService.java [hunk 3]
```diff
@@ -203,7 +206,7 @@ public void updateQuickTestWithPersonalData(Map<String, String> ids, String shor
     public void removeAllBefore(LocalDateTime deleteTimestamp) {
         quickTestRepository.findAllByCreatedAtBeforeAndVersionIsGreaterThan(deleteTimestamp, 0).forEach(quickTest -> {
             this.sendResultToTestResultServer(quickTest.getTestResultServerHash(),
-                TestResult.FAILED.getValue(),
+                TestResult.FAILED.getValue(), deleteTimestamp.toEpochSecond(ZoneOffset.UTC),
```
Comment (mschulte-tsi) [inline src/main/java/app/coronawarn/quicktest/service/QuickTestService.java:209]: move added code to a new line

File: src/main/java/app/coronawarn/quicktest/service/QuickTestService.java [hunk 4]
```diff
@@ -147,6 +148,7 @@ public void updateQuickTest(Map<String, String> ids, String shortHash, short res
             throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR);
         }
         sendResultToTestResultServer(quicktest.getTestResultServerHash(), result,
+                quicktest.getUpdatedAt().toEpochSecond(ZoneOffset.UTC),
```
Comment (mschulte-tsi) [inline src/main/java/app/coronawarn/quicktest/service/QuickTestService.java:151]: set indention to same level for same method

File: src/main/java/app/coronawarn/quicktest/service/QuickTestService.java [hunk 5]
```diff
@@ -188,6 +190,7 @@ public void updateQuickTestWithPersonalData(Map<String, String> ids, String shor
             throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR);
         }
         sendResultToTestResultServer(quicktest.getTestResultServerHash(), quicktest.getTestResult(),
+                quicktest.getUpdatedAt().toEpochSecond(ZoneOffset.UTC),
```
Comment (mschulte-tsi) [inline src/main/java/app/coronawarn/quicktest/service/QuickTestService.java:193]: set indention to same level for same method

File: src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java (hunk 2 above)
Comment (mschulte-tsi) [reply src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java:35]: both added newlines

File: src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java (hunk 1 above)
Comment (mschulte-tsi) [reply src/main/java/app/coronawarn/quicktest/model/QuickTestResult.java:23]: both added newlines

**Review by mschulte-tsi — CHANGES_REQUESTED** [review]
Please adapt

**Review by mschulte-tsi — APPROVED** [review]
LGTM

**Pull request merged**